exhale/configs.py

```diff
@@ -1505,6 +1522,7 @@ def apply_sphinx_configurations(app):
         ("treeViewBootstrapLevels",                      int),
         # Page Level Customization
         ("includeTemplateParamOrderList",               bool),
+        ("fullyQualifiedTitles",                        bool),
```

Comment:
there is another option that comes to mind, three modes: "default", "never", "always".

- default: use shorter name if no overloaded name exists. that data is parsed elsewhere, I would be able to help with this.
- never: always use short title
- always: always use fully qualified title

I'm OK with just keeping it "on/off" for simplicity, but figured I'd see what your thoughts are on it before we continue forward.

Reply:
This seems a nice extension, provided that the TOC can stay sorted consistently, e.g.

 * Struct abc (in ns1::)
 * Struct ns1::foobar
 * Struct cde (in ns2::)
 * Struct ns2::foobar

Otherwise I fear it might be hard to browse. However the order of appearance should be set by exhale, correct?

Concerning the overloaded names, is such data available (specifically from `initializeNodeFilenameAndLink`)? If you can point me to that I can try out some change. Perhaps we should refer to this as "clashing names", since "overloads" in C++ refer to something else (and also wouldn't be able to identify structures with the same name in different namespaces).